Storage/SQL/SQLiteHistory.swift

```diff
@@ -8,6 +8,8 @@ import XCGLogger
 
 private let log = Logger.syncLogger
 
+private let TopSitesCacheSize = 100
```

Comment:
Notably: the smaller the device, the suckier it is, so we do care about making this as small as we can.